deploy/coins/ethereum/scripts/entry-dev.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+geth --dev --dev.period 5 --verbosity 2 --networkid 15 --datadir "data" -mine --miner.threads 1 -http --http.addr 0.0.0.0 --http.port 8545 --allow-insecure-unlock --http.api "eth,net,web3,miner,personal,txpool,debug" --http.corsdomain "*" -nodiscover --http.vhosts="*"
```

Comment:
Missing NL.

What's the provenance on 15? I see it on a StackExchange Q but no further details. It doesn't appear to conflict with anything relevant but hardhat and foundry use 31337 for development purposes, so I'd rather use that which is more acknowledged.

Also, this seems fine for a dev environment, yet in general seems to be extremely accessible regarding bind + cors + etc. I know its a dev env, behind Docker, yet still would appreciate if it can be tightened.